src/components/Navigation.js

```diff
@@ -0,0 +1,61 @@
+import React, {useState} from "react";
+import {NavLink} from "react-router-dom";
+import LoginPage from "../pages/LoginPage";
+
+function Navigation() {
+    const [isAuth, setIsAuth] = React.useState(false);
+
+    let linkText = "Inloggen";
+
+    if (isAuth) {
+        linkText = "Uitloggen"
+    }
+
+    return (
+        <nav>
+            <div className="nav-container">
+                <ul>
+                    <NavLink to="/" exact activeClassName="active-link">
+                        <li>
+                            Home
+                        </li>
+                    </NavLink>
+
+                    <NavLink to="/login" exact activeClassName="active-link">
+                        <li>
+                            {linkText}
+                        </li>
+                    </NavLink>
+
+                    {isAuth &&
+                    <NavLink to="/blogposts" exact activeClassName="active-link">
+                        <li>
+                            Blog Overzicht
+                        </li>
+                    </NavLink>
+                    }
+
+                    {isAuth &&
```

Comment:
In de uitwerkingen hebben ze {isAuth===true ? <>etc.

Maar ik vind wat jij hebt gebruikt {isAuth &&.....etc. compacter ogen.